packages/credo/src/resource.ts

```diff
@@ -351,40 +351,6 @@ export class ResourceHandler {
 				}
 			}
 		);
-		// Register a resource to report the status of Credo Agent
-		server.resource('credo-agent-status', 'status://agent/credo', async (uri) => {
```

Comment:
Removed status from resource and added as a function which remote-server can expose as endpoint